src/components/Header/index.jsx

```diff
@@ -37,7 +37,7 @@ class Header extends React.Component {
           {email && <Dropdown
             title={email}
             menuItems={[
-              <LinkWrapper to="/logout">Logout</LinkWrapper>,
```

Comment:
Now that we no longer need to use the `LinkWrapper` component, we might want to remove the component entirely; this was the only place it was being used.

Reply:
Does `LinkWrapper` provide something that we don't get from an `<a>` tag?

Reply:
@douglashall The links within the Paragon `Dropdown` component must be react-router `Link` components for routing purposes. However, there was an issue with just using the standard `Link` since the `ref` gets set with an `innerRef` prop, not the usual `ref` prop. The `LinkWrapper` component was just that -- a wrapper around `Link` that a `ref` can be set on with an associated `focus` method.

Reply:
Okay cool, so since we are not linking to another page within the SPA we don't need `Link` or `LinkWrapper`.